internal/tool/multi/deploy.go

```diff
@@ -84,6 +87,37 @@ func deploy(ctx context.Context, args []string) error {
 	}
 	multiConfig.App = appConfig
 
+	// Check version compatibility.
+	versions, err := bin.ReadVersions(appConfig.Binary)
+	if err != nil {
+		return fmt.Errorf("read versions: %w", err)
+	}
+	if versions.DeployerVersion != version.DeployerVersion {
+		// Try to relativize the binary, defaulting to the absolute path if
+		// there are any errors..
+		binary := appConfig.Binary
+		if cwd, err := os.Getwd(); err == nil {
+			if rel, err := filepath.Rel(cwd, appConfig.Binary); err == nil {
+				binary = rel
+			}
+		}
+		return fmt.Errorf(`ERROR: The binary you're trying to deploy (%q) was built with
+github.com/ServiceWeaver/weaver module version %s (deployer API
```

Comment:
Remove all references of `deployer API`. They should just know that version of code and version of `weaver` are incompatible.

Reply:
Done.